certbot-auto-renewal.sh

```
@@ -0,0 +1,50 @@
+mkdir -p "$LOG_DIR"
+COMBINED_LOG="$LOG_DIR/certbot-renew-combined-$NOW.log"
+EMAIL_FILE="$LOG_DIR/email-combined-$NOW.txt"
+
+# === Start Combined Log ===
+echo "=== Certbot Renewal Started at $(date) ===" > "$COMBINED_LOG"
+
+# === Process Each Cert ===
+for CERT_NAME in "${CERT_NAMES[@]}"; do
+    echo -e "\n--- Renewing $CERT_NAME at $(date) ---\n" >> "$COMBINED_LOG"
+
+    sudo certbot run --force-renew --redirect -d "$CERT_NAME" --verbose 2>&1 >> "$COMBINED_LOG"
```

Shellcheck (0.10.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)